src/lib/Handset/CRSFHandset.h

```diff
@@ -21,14 +21,15 @@ class CRSFHandset final : public Handset
     void End() override;
 
 #ifdef CRSF_TX_MODULE
-    bool IsArmed() override { return CRSF_to_BIT(ChannelData[4]); } // AUX1
+    bool IsArmed() override { return armCmd; } // AUX1/CH5 (Channel mode) or extra byte in RC channesl message (Function mode)
     void handleInput() override;
     void handleOutput(int receivedBytes);
 
     static HardwareSerial Port;
     static Stream *PortSecondary; // A second UART used to mirror telemetry out on the TX, not read from
 
     static uint8_t modelId;         // The model ID as received from the Transmitter
+    bool armCmd;                    // Arm command from handset either via ch5 or arm message
```

Comment:
oops for forgot one, this should be private

Reply:
done